atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -1790,6 +1779,16 @@ private void putMetadataAndMaybeAlterTables(
         }
     }
 
+    private String schemaChangeDescriptionForPutMetadataForTables(Collection<CfDef> updatedCfs) {
+        String tables = updatedCfs.stream()
+                .map(CassandraKeyValueServices::tableReferenceFromCfDef)
+                .map(Object::toString)
+                .collect(Collectors.toList())
+                .toString();
+        return String.format("after updating the column family for tables %s in a call to put metadata for tables",
+                tables);
```

Comment:
This is not generally speaking going to be safe for logging. I don't think we log the schema change description anywhere, but it might be good if we name the param in `waitForSchemaVersions` `unsafeSchemaChangeDescription` or something like that.